katello_certs_tools/sslToolConfig.py

```diff
@@ -522,11 +515,11 @@ def figureSerial(caCertFilename, serialFilename, indexFilename):
     # create the serial file if it doesn't exist
     # write the digits to this file
     open(serialFilename, 'w').write(serial+'\n')
-    os.chmod(serialFilename, 0600)
+    os.chmod(serialFilename, '0600')
 
     # truncate the index.txt file. Less likely to have unneccessary clashes.
     open(indexFilename, 'w')
-    os.chmod(indexFilename, 0600)
+    os.chmod(indexFilename, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)